### PR TITLE
Story/io 156/show category in planning list view

### PR DESCRIPTION
**Pull request by EngbergJonas**
- Added category tag to planning table project header 
- Removed icons not to be used in MVP

### CODE REVIEW

**Comment by EngbergJonas** [comment]
Should be merged after https://github.com/City-of-Helsinki/infraohjelmointi-ui/pull/26.

File: src/components/shared/CustomTag.tsx
Comment (kgunga) [inline src/components/shared/CustomTag.tsx:1]: wrap with memo because component is receiving properties

**Review by kgunga — CHANGES_REQUESTED** [review]

File: src/components/shared/CustomTag.tsx
Comment (EngbergJonas) [reply src/components/shared/CustomTag.tsx:1]: fixed

**Review by kgunga — APPROVED** [review]
👍

**Pull request merged**